index.html

```diff
@@ -93,22 +93,40 @@
       </p>
 
       <p>
-This document specifies the interface and guidelines for the DID 
+This document specifies the interface and guidelines for the DID
 resolution process.
       </p>
     </section>
 
-<section id='sotd'>
-<p>
-Comments regarding this document are welcome. Please file issues
-directly on <a href="https://github.com/w3c-ccg/did-resolution/issues/">GitHub</a>,
-or send them to
-<a href="mailto:public-credentials@w3.org">public-credentials@w3.org</a>
-(<a href="mailto:public-credentials-request@w3.org?subject=subscribe">subscribe</a>,
-<a href="https://lists.w3.org/Archives/Public/public-credentials/">archives</a>).
-</p>
+    <section id='sotd'>
+    <p>
+    Comments regarding this document are welcome. Please file issues
+    directly on <a href="https://github.com/w3c-ccg/did-resolution/issues/">GitHub</a>,
+    or send them to
+    <a href="mailto:public-credentials@w3.org">public-credentials@w3.org</a>
+    (<a href="mailto:public-credentials-request@w3.org?subject=subscribe">subscribe</a>,
+    <a href="https://lists.w3.org/Archives/Public/public-credentials/">archives</a>).
+    </p>
+
+    <p>
+    Portions of the work on this specification have been funded by the
+    United States Department of Homeland Security's Science and Technology
+    Directorate under contracts HSHQDC-16-R00012-H-SB2016-1-002 and
+    HSHQDC-17-C-00019. The content of this specification does not necessarily
+    reflect the position or the policy of the U.S. Government and no official
+    endorsement should be inferred.
+    </p>
+
+    <p class="issue">
+    Names in this section need to be updated:
+
+    Work on this specification has also been supported by the Rebooting the
+    Web of Trust community facilitated by Christopher Allen, Shannon Appelcline,
+    Kiara Robles, Brian Weller, Betty Dhamers, Kaliya Young, Manu Sporny,
```

Comment:
The most up-to-date list is here:

Work on this specification has also been supported by the Rebooting the Web of Trust community facilitated by Christopher Allen, Shannon Appelcline, Kiara Robles, Brian Weller, Betty Dhamers, Kaliya Young, Kim Hamilton Duffy, Manu Sporny, Drummond Reed, Joe Andrieu, and Heather Vescent.